src/database/seeds/DatabaseSeeder.php

```diff
@@ -11,6 +11,8 @@ class DatabaseSeeder extends Seeder
      */
     public function run()
     {
-        // $this->call(UsersTableSeeder::class);
+        $this->call(BigQuestionsTableSeeder::class);
```

Comment:
まとめて表現もできます。
どちらでも可、メンテナンスしやすいのはまとめる方かなと個人的に思います
```suggestion
        $this->call([BigQuestionsTableSeeder::class, QuestionsSeeder::class]);
```